hello.erl

```diff
@@ -0,0 +1,3 @@
+-module(hello).
+	-export([hello_world/0]).
+	hello_world() -> io:fwrite("hello world! \n").
```

Comment:
@fatemeh123 what about capitalizing the first letter in word "hello"? 
Nothing serious! But just a beauty tip!